src/camera_session.cpp

```diff
@@ -50,6 +56,13 @@ int CameraSession::init() {
     return 0;
   }
 
+  validateConfig();
```

Comment:
We need to narrow this down to one validateConfig, we will validate a second time in some cases, need to be wary of performance.

Reply:
I agree. I've removed the second validateConfig. Let me know what you think when you get a chance